mobile/native/migrations/2024-04-09-105312_ensure_old_orders_have_matching_fee/up.sql

```diff
@@ -0,0 +1,17 @@
+-- Your SQL goes here
+update orders
+set matching_fee_sats = quantity / (execution_price * leverage) * 100000000
```

Comment:
🔧 I think this is missing the fee_rate

```suggestion
set matching_fee_sats = quantity / (execution_price * leverage) * 100000000 * 0.003
```

Reply:
Makes sense 🙈